src/backend/base/langflow/services/utils.py

```diff
@@ -157,17 +164,92 @@ def initialize_session_service() -> None:
     )
 
 
+async def clean_transactions(settings_service: "SettingsService", session: AsyncSession) -> None:
```

Comment:
Can the quotes on `"SettingsService"` be removed ? (may need to add `from __future__ import annotations`)

Reply:
Yes. Thanks!